profiles/arch/x86/use.mask

```diff
@@ -181,6 +181,9 @@ webengine
 -video_cards_vesa
 -video_cards_qxl
 
+# unmask video_cards with virtgpu DRM native-context support
+-video_cards_asahi
```

Comment:
Do we really need to expose this on x86?

Note that we don't need to unmask it here for 32-bit multilib support on amd64.

Reply:
Yes. We must build the driver for x86 and amd64 to enable its use in our x86 translation stack.